staticcheck/testdata/src/CheckByteSliceInIOWriteString/CheckByteSliceInIOWriteString.go

```diff
@@ -0,0 +1,16 @@
+package pkg
```

Comment:
We should also add tests for the cases that we _don't_ want to flag. WriteString with a string, with a string(b) where b is not a []byte.

We should also add tests for more cases we do want to flag, such as string(b) where b's type is T and `type T []byte`.